src/H5PB.c

```diff
@@ -1302,6 +1302,73 @@ H5PB_write(H5F_shared_t *f_sh, H5FD_mem_t type, haddr_t addr, size_t size, const
     FUNC_LEAVE_NOAPI(ret_value)
 } /* end H5PB_write() */
 
+/*-------------------------------------------------------------------------
+ * Function:    H5PB_enabled
+ *
+ * Purpose:     Check if the page buffer may be enabled for the specified
+ *              file and data access type.
+ *
+ * Return:      Non-negative on success/Negative on failure
+ *
+ * Programmer:  Neil Fortner
+ *
+ *-------------------------------------------------------------------------
+ */
+htri_t
+H5PB_enabled(H5F_shared_t *f_sh, H5FD_mem_t type)
+{
+    H5PB_t *page_buf;          /* Page buffering info for this file */
+    hbool_t bypass_pb = FALSE; /* Whether to bypass page buffering */
+    htri_t  ret_value;         /* Return value */
```

Comment:
htri_t is a hack, especially now that C has a real Boolean type. An herr_t return value with an hbool_t out parameter would be better.